<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="random_checkbox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>78</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
